src/main/java/network/brightspots/rcv/StreamingCvrReader.java

```diff
@@ -227,15 +232,15 @@ private void cvrCell(int col, String cellData) {
       if (!isNullOrBlank(overvoteDelimiter)) {
         candidates = cellString.split(Pattern.quote(overvoteDelimiter));
       } else {
-        candidates = new String[] {cellString};
+        candidates = new String[]{cellString};
       }
 
       for (String candidate : candidates) {
         candidate = candidate.trim();
-        if (candidates.length > 1 && (candidate.equals("") || candidate.equals(skippedRankLabel))) {
+        if (candidates.length > 1 && (candidate.isBlank() || candidate.equals(skippedRankLabel))) {
```

Comment:
does this change have any practical consequence?

Reply:
No, because of the `trim` above, but this is different in that a series of whitespace `isBlank` but is not `equals("")`